app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+    def remove_animal(self) -> None:
+        if self in Animal.alive:
+            Animal.alive.remove(self)
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
```

Comment:
You can rewrite this method in one line )

Reply:
yeah, fix that)